txsni/test/certs/DEFAULT.pem

```diff
@@ -0,0 +1,45 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
On second thought, I don't want to get email from security scanners about this :).  Can these keys be generated on the fly (or at least "once per developer") rather than checked in?